### ISSUE TITLE
dvc queue logs -f could take more then one job_id

### ISSUE THREAD

**Issue by behrica** [iterative/dvc#8658]
Similar to "dvc exp run --run-all", which follows the log of all running jobs.

so maybe as simple as:

```
dvc queue logs -f 107f918,a6ca492
```

or maybe as options to follow "all running"

```
dvc queue logs -f all
```

**Comment by dberenbaum** [comment]
It could be as simple as `dvc queue logs` without any args follows all tasks.

**Comment by dberenbaum** [comment]
This would also be useful as a replacement for `dvc exp run --run-all`. See https://github.com/iterative/dvc/discussions/8724.

**Comment by ReveStobinson** [comment]
I agree with this, especially given that `dvc exp run --run-all` will be deprecated in a future release! The documentation says:

> [dvc exp run --run-all [--jobs]](https://dvc.org/doc/command-reference/exp/run#--run-all) is now a shortcut for [dvc queue start [--jobs]](https://dvc.org/doc/command-reference/queue/start#--jobs) followed by [dvc queue logs -f](https://dvc.org/doc/command-reference/queue/logs#-f). 

However, there isn't actually a set of suitable DVC commands that could replace `--run-all` without the addition of _at least_ the option suggested by @behrica, since `dvc queue logs -f` requires exactly one task to be specified. 

In that spirit, I would go even further and suggest that an option should also be added to `dvc queue start` (perhaps `--logs`, `--follow`, or `--attach`), which follows the logs of all of the experiments in the queue (for which I imagine the easiest implementation would be creating a `--follow-all` option for `queue logs` and making it a shortcut).

**Comment by dberenbaum** [comment]
Let's make `dvc queue logs` show logs for all experiments by default, and then make `dvc queue start --follow` equivalent to `dvc queue start && dvc queue logs --follow`.

**Comment by BradyJ27** [comment]
Afaik `dvc queue logs` does NOT show all experiments by default still, and this issue is still valid. Is that correct @dberenbaum ?

**Comment by dberenbaum** [comment]
Yes, that's correct.
